### ISSUE TITLE
Events API returning wrong results

### ISSUE THREAD

**Issue by anuveyatsu** [datahq/metastore#19]
It seems like a bug as it is returning result for `co2-fossil-global` as well, however, I am requesting `co2-ppm`: http://api.datahub.io/metastore/search/events?owner=%22core%22&dataset=%22co2-ppm%22

Also it returns results even if I pass only part of dataset name, e.g., `dataset="co2"`.

**Comment by rufuspollock** [comment]
@zelima did you see this? Can you comment ...

**Comment by zelima** [comment]
@rufuspollock yes I've seen. I have not investigated this yet. Probably next sprint

**Comment by zelima** [comment]
Hm... actually I just reindexed testing with proper analyzers (without changing `match` query to `terms` query) and the result is same. https://api-testing.datahub.io/metastore/search/events?owner=%22core%22&dataset=%22co2-ppm%22

The only difference now is that it's case sensitive
